apps/test/unit/code-studio/pd/professional_learning_landing/landingPageTest.js

```diff
@@ -20,12 +20,13 @@ describe('Tests for Professional Learning Landing Page', () => {
         deeperLearningCourseData: [{data: 'oh yeah'}],
       });
 
-      expect(landingPage.childAt(2).is('LastWorkshopSurveyBanner')).to.be.true;
-      expect(landingPage.childAt(2).prop('subHeading')).to.equal(
+      const mainWrapper = landingPage.find('main');
```

Comment:
Needed to add this `.find()` method to all of these tests since I added the `<main>` wrapper in `LandingPage` component.